src/middleware/yarp/fixture-ymanager/YarpFixManager.cpp

```diff
@@ -61,7 +61,7 @@ bool YarpFixManager::setup(int argc, char** argv) {
         enableRestrictedMode();
 
         // load the fixture (application xml)
-        char szAppName[255];
+        char* szAppName = new char [10];
```

Comment:
You are the expert! :stuck_out_tongue_closed_eyes: 
I'm just worried that these 10 bytes are leaked somewhere...